src/SWP/Bundle/ContentBundle/Resources/config/services.yml

```diff
@@ -31,6 +31,11 @@ services:
         tags:
             - { name: kernel.event_listener, event: kernel.request, method: onKernelRequest }
 
+    swp_content_bundle.listener.http_cache_header:
+        class: SWP\Bundle\ContentBundle\EventListener\HttpCacheHeaderListener
```

Comment:
put `SWP\Bundle\ContentBundle\EventListener\HttpCacheHeaderListener` into the `parameters` section in this file and just put the reference to that parameter here, so it will be easier to override that class in the future, for example by 3rd parties when needed.

```yaml
parameters:
    swp_content_bundle.listener.http_cache_header.class: SWP\Bundle\ContentBundle\EventListener\HttpCacheHeaderListener

swp_content_bundle.listener.http_cache_header:
    class: '%swp_content_bundle.listener.http_cache_header.class%'
```